README.md

```diff
@@ -1 +1,32 @@
-# spitoruler
\ No newline at end of file
+# SPITO
+SPTIO (Simple Powerful Interprocess Tool to Overpower) has been designed
+to allow users easily swap and edit configuration files.
+
+## Use cases
+**For a while it's a myth language**
+
+### Simple config
+- used rule: samba-guest-are-god
+- purpose: use whole ready-to-use samba config
+```shell
+spito -r samba-guests-are-god
```

Comment:
We won't have anything like `--replace` / `-r` because SPITO uses rules in order to anything instead of copy pasting configs

Reply:
I agree